module/platform/darwin/HardwareIO/tests/ConversionTest.cpp

```diff
@@ -173,42 +174,42 @@ TEST_CASE("Testing the hardware position conversions to radians", "[hardware][co
     {
         INFO("Testing the inverse position conversions");
 
-        for (size_t i = 0; i < 20; ++i) {
+        for (ssize_t i = 0; i < 20; ++i) {
```

Comment:
I'd really just use int here, no need for ssize_t when an int will do the trick. It's just one more thing for people to need to understand to read this code